### DISCUSSION TITLE
Appearance settings

### DISCUSSION THREAD

**Discussion by franco-ajfpa** [sepinf-inc/IPED#1717] [General]
How can I increase the resolution of IPED 3.18.14 (increase the size of the characters)?

**Comment by lfcnassif** [comment]
AFAIK you can focus on the table and type Ctrl+/Ctrl- to increase/decrease characters size. It doesn't resize all UI components and it doesn't apply to texts of some Dialogs. In 4.0 and later versions that was replaced by @tc-wleite by a much better UI scaling feature that scales all UI components, which is located in the Options button -> Layout/Appearance submenu.

**Reply by franco-ajfpa** [reply]
thank you
